lib/browser/modules/selection-state.js

```diff
@@ -124,15 +124,26 @@ selectionState.service('SelectionStateService', function() {
   };
 
   /**
-   * @summary Clear all selections
+   * @summary Clear selections based on parameter passed
    * @function
    * @public
    *
+   * @param {Object} options - options
+   *
    * @example
    * SelectionStateService.clear();
+   *
+   * @example
+   * SelectionStateService.clear(options);
    */
-  this.clear = function() {
-    selection = {};
+  this.clear = function(options) {
```

Comment:
The new behaviour of this function should be unit tested in `tests/browser/modules/selection-state.spec.js`: https://github.com/resin-io/resin-etcher/blob/master/tests/browser/modules/selection-state.spec.js#L154.